### ISSUE TITLE
Random crashes

### ISSUE THREAD

**Issue by DjSni** [fabianishere/pve-edge-kernel#259]
I can't say for sure, but I have crashes with the kernel 5.16 and 5.17 about every 1 - 3 days. I always keep the versions up to date - I'm currently on 5.17.0-edge. Sometimes the whole thing goes so far that the CPU fan turns very high, but nothing reacts anymore (no ping, like dead). The only thing that helps is to de-energise the computer. I am currently using a Minisforum Tiny (HX90) as my node, which has a built-in APU AMD Ryzen 9 5900HX.
Unfortunately I don't get any kernel panic logs that I could attach here.

**Comment by fabianishere** [comment]
Are you using ZFS as well?

**Comment by DjSni** [comment]
No

**Comment by fabianishere** [comment]
I am investigating a similar issue with an Intel-based machine showing the same symptoms since Linux 5.14, where the kernel seems to be deadlocking somehow.

I originally suspected ZFS, due to it being the only machine in my cluster that uses it. My solution for that machine is to run the Proxmox 5.13 kernel.

**Comment by DjSni** [comment]
That's exactly how it feels, that the kernel blocks everything. Well, I'll try the Proxmox 5.13 kernel, but then it's probably a general kernel problem.

**Comment by DjSni** [comment]
> @DjSni appreciate though that's not helpful in your case, as you don't have an iGPU at all, especially not an intel one 😅

Thanks for the info, but my APU has an iGPU, it's actually a notebook processor. I have an AMD Ryzen 9 5900HX with Radeon Graphics and not to be confused with the Ryzen 9 5900X which has no iGPU. Unfortunately, turning off the iGPU is not an option for me as I currently need it for Emby.

**Comment by fabianishere** [comment]
I have updated the machine to Linux 5.18 since then and haven't seen these issues anymore.

**Comment by DjSni** [comment]
In the meantime I changed the computer, there I had random reboots and crashes like before.
I was able to find out that this is probably due to the combination RAM and C6 sleep stats are not compatible.
In the BIOS the "Power Down Enable" option must be set to Disable because the C6 state is unstable with DRAM "Power Down Enable".
I hope this helps others who have or had the same problem.

P.S.: I am now on the 5.19 kernel and haven't had a single reboot or crash.
